src/pages/workspaces/workspace/Data.js

```diff
@@ -501,6 +502,11 @@ const WorkspaceData = _.flow(
                     selected: _.isEqual(selectedDataType, [snapshotName, tableName]),
                     onClick: () => {
                       setSelectedDataType([snapshotName, tableName])
+                      Ajax().Metrics.captureEvent(Events.workspaceSnapshotContentsView, {
+                        ...extractWorkspaceDetails({ workspace }),
+                        snapshotId: referenceId,
+                        entityType: tableName
+                      })
```

Comment:
nit: could you rename the `snapshotId` property to `referenceId`, since this is actually the reference's uuid? If possible, could you also include the underlying data repo snapshot's id?

Given this payload:
```
{
  "cloningInstructions": "COPY_NOTHING",
  "description": "",
  "name": "snappy",
  "reference": {
    "instanceName": "terra",
    "snapshot": "cf0e417d-1456-401d-bbf0-a3fa2bd3afa3"
  },
  "referenceId": "34542b10-e69a-4f02-84c2-6b4b8c6fc852",
  "referenceType": "DATA_REPO_SNAPSHOT",
  "workspaceId": "60bd54fe-5ca2-4763-b9f5-03a0a5083f89"
}
```

the ideal would be to include
```
referenceId: "34542b10-e69a-4f02-84c2-6b4b8c6fc852",
snapshotId: "cf0e417d-1456-401d-bbf0-a3fa2bd3afa3"
```

in the event.